src/components/App.js

```diff
@@ -23,9 +23,12 @@ export class App extends Component {
         <Route exact path='/login' component={LoginForm} />
         <Route
           exact
-          path='/movie/:id'
+          path='/movies/:id'
           render={({ match }) => {
-            let moviesData = [...this.props.movies];
+            const { id } = match.params;
+            let moviesData = this.props.allMovies.find(
+              movie => movie.id === parseInt(id)
+            );
```

Comment:
Would there be a way to pull them from the Redux store directly into the MovieShowPage component rather than passing through directly as props from App?  Something Robbie said in class today in response to Matt made me think that is how we would do this.
Since we need to use the match.params, I'm not clear exactly how that would work.  Can we look at it together tomorrow for possible refactoring?